Lang/Spanish.pj.Lang

```diff
@@ -177,7 +178,7 @@
 #320#  "Seleccionar directorio de ROMs actual"
 
 //Messages
-#340#  "¿ROM Dañado? Usa GoodN64 y verifica que la RDB esté actualizada."
+#340#  "¿ROM Dañada? Usa GoodN64 y comprueba por una RDB actualizada."

```

Comment:
> You can change this translation without needing to modify the English version.

yes, but that is not right for a proper translation, in a proper translation you have to preserve the original context of the original lang.

about "condescending" the right spanish translation is "condescendiendo", while "condescendent" is "condescendiente".